spec/mongoid/copyable_spec.rb

```diff
@@ -54,6 +54,30 @@
         end
       end
 
+      context "when using store_as" do
+        context "and dynamic attributes are not set" do
+          it "clones" do
+            class StoreAsDupTest1
```

Comment:
I think it is better to define this classes outside this test, as this is in a loop.